package.json

```diff
@@ -40,15 +40,16 @@
     "glob": "7.0.6",
     "html-wiring": "1.2.0",
     "insight": "0.8.3",
+    "jhipster-core": "1.2.0",
+    "js-yaml": "3.6.1",
     "lodash": "4.15.0",
     "mkdirp": "0.5.1",
-    "shelljs": "0.7.4",
     "pluralize": "3.0.0",
-    "yeoman-generator": "0.24.1",
-    "js-yaml": "3.6.1",
-    "yo": "1.8.5",
     "semver": "5.3.0",
-    "jhipster-core": "1.2.0"
+    "shelljs": "0.7.4",
+    "uuid": "3.0.0",
```

Comment:
Here I simply added a dependency on the uuid lib but npm reordered them for me.
I need UUID to setup sample alerting webhook on [http://webhook.site](http://webhook.site).
I will add something similar for ELK soon.